Document/0x05b-Basic-Security_Testing.md

```diff
@@ -623,17 +623,19 @@ More information on using the Objection REPL can be found on the [Objection Wiki
+[radare2](https://rada.re/r/ "Radare2 official website") (r2) is a popular open source reverse engineering framework for disassembling, debugging, patching and analyzing binaries that is scriptable and supports [many architectures and file formats](https://rada.re/r/cmp "radare2 Comparison Table") including Android/iOS apps. For Android, Dalvik DEX (odex, multidex), ELF (executables, .so, ART) and Java (JNI and Java classes) are supported. It also contains several useful scripts that can help you during mobile application analysis as it offers low level disassembling and safe static analysis that comes handy when traditional tools fail.
+
+radare2 implements a rich command line interface (CLI) where you can perform the mentioned tasks. However, if you're not really comfortable using the CLI for reverse engineering you may want to consider using the Web UI (via the `-H` flag) or the even more convenient Qt and C++ GUI version called [Cutter](https://github.com/radareorg/cutter "Cutter"). However, keep in mind that the CLI, and more concretely its Visual Mode and its scripting capabilities ([r2pipe](https://github.com/radare/radare2-r2pipe "r2pipe")), are the core of radare2's power and it's definitely worth learning how to use it.
 
 ###### Installing radare2
 
-Please refer to [radare2's official installation instructions](https://github.com/radare/radare2/blob/master/README.md "radare2 installation instructions").
+Please refer to [radare2's official installation instructions](https://github.com/radare/radare2/blob/master/README.md "radare2 installation instructions"). We highly recommend to install radare2 always from the GitHub version instead of via common packet managers such as APT, as the version in their repositories might be probably already outdated. Radare2 is in active development, every day you'll find changes, therefore it’s recommended to keep it updated to the latest version via git.
 
 ###### Using radare2
 
-r2 is a set of small command-line utilities that can be used together or independently. Utilities like `rabin2`, `rasm2`, `rahash2`, `radiff2`, `rafind2`, `ragg2`, `rarun2`, `rax2`, and of course `radare2`.
+The radare2 framework comprises a set of small utilities that can be used from the r2 shell or independently as CLI tools. These utilities include `rabin2`, `rasm2`, `rahash2`, `radiff2`, `rafind2`, `ragg2`, `rarun2`, `rax2`, and of course `r2`, which is the main one.
 
-You can use `rafind2` to read strings directly from the encoded AndroidManifest.xml. Use this to list permissions, activities, providers, services, receivers and other information stored in AndroidManifest.xml
+For example, you can use `rafind2` to read strings directly from the encoded in an Android Manifest (AndroidManifest.xml). Use this to list permissions, activities, providers, services, receivers and other information stored in the Android Manifest.
```

Comment:
from the encoded in an Android Manifest (AndroidManifest.xml)  ->  from an encoded AndroidManifest.xml.

Reply:
I thought maybe the second sentence is redundant (one can just read the list in the code block) so I removed it and wrote: 

For example, you can use `rafind2` to read strings directly from an encoded Android Manifest (AndroidManifest.xml):